docs/user-guide/configure-tls.md

```diff
@@ -1,20 +1,19 @@
 # Configuring Transport Layer Security
 
-Transport Layer Security (TLS) can be enabled on Styx network
-interfaces. You can enable it on the proxy server port, and also for
-each of the backend services, where necessary.
+Transport Layer Security (TLS) can be enabled on Styx both for the incoming interface
+(connections from clients) and the connections with the backend services.
 
-TLS can be enabled independently for each network interface. You can
-enable it independently for proxy server port, and for each of the
-back-end services. Styx will convert between the protocols where
+TLS can be enabled independently for each network resource. You can
```

Comment:
The usage of term “network resource” is rether confusing. What we are trying to say is that the Styx server port and each app can have their TLS settings configured separately.

Reply:
Ok.
The document used to say "for each network interface" which, to me, would mean that you can bind to different network cards / ip addresses.  I don't see any part of the code that allows for this.  If it's some feature we have lost or I've missed it, please tell me so.

Reply:
You are correct. The original use of "network interface" was also a poor choice of words. But it reads much better now for what I can see 👍